charts/argo/templates/server-deployment.yaml

```diff
@@ -67,16 +67,20 @@ spec:
             value: {{ .Values.server.baseHref | quote }}
           resources:
             {{- toYaml .Values.server.resources | nindent 12 }}
-          {{- with .Values.server.volumeMounts }}
           volumeMounts:
+          - name: tmp
```

Comment:
I assume this is always `/tmp` and the temp path cannot change? i.e. is it controllable in Argo?

Reply:
No, it's [hard-coded](https://github.com/argoproj/argo/pull/4350/files#r511150735).